docs/importing_astropy.rst

```diff
@@ -1,16 +1,16 @@
-*********************************
-Importing astropy and subpackages
-*********************************
+************************************
+Importing `astropy` and Sub-packages
```

Comment:
When astropy (lowercase) appears in a title like this, I think you have to do 2 backticks: 
```suggestion
Importing ``astropy`` and Sub-packages
```